pom.xml

```diff
@@ -19,7 +19,7 @@
     <http.port>8081</http.port>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${ramlfiles_path}/raml-util</ramlfiles_util_path>
-    <raml-module-builder.version>34.0.0</raml-module-builder.version>
+    <raml-module-builder.version>35.0.0</raml-module-builder.version> <!-- also update vertx.version -->
```

Comment:
RMB 35.0.1 has been released, we may want to bump the version here prior to merge